docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -48,6 +48,41 @@ for test code. The tasks for custom source sets are named according to their `co
 
 The names of the tasks in Android Projects contain [build variant](https://developer.android.com/studio/build/build-variants.html) names and follow the `compile<BuildVariant>Kotlin` pattern, for example, `compileDebugKotlin` or `compileReleaseUnitTestKotlin`.
 
+For both JVM and Android Projects it's possible since 1.9.0 to define options in a following way:
```

Comment:
```suggestion
For both JVM and Android Projects it's possible since 1.9.0 to define options in a following way using project Kotlin extension DSL:
```